### PR TITLE
[libexif] Add processing of comment tags used by Windows.

### PR DESCRIPTION
**Pull request by ace20022**
Feature requested here: http://trac.kodi.tv/ticket/15936

EDIT: tag list: http://www.exiv2.org/tags.html

### CODE REVIEW

File: lib/libexif/ExifParse.cpp [hunk 1]
```diff
@@ -480,6 +481,15 @@ void CExifParse::ProcessDir(const unsigned char* const DirStart,
       }
       break;
 
+      case TAG_XP_COMMENT:
+      {
+        // The XP user comment is allows unicode encoded
```
Comment (ksooo) [inline lib/libexif/ExifParse.cpp]: This sentence sounds strange to me. Did you perhaps mean "always" instead of "allows"?

Comment (ace20022) [inline lib/libexif/ExifParse.cpp]: ofc, c&p fail from above ;) thx!

**Comment by MartijnKaijser** [comment]
could you rebase?

**Comment by ace20022** [comment]
Sure, next week

**Comment by ace20022** [comment]
jenkins build this please

**Pull request merged**